routing/pathfind.go

```diff
@@ -605,6 +609,10 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 
 		// Advance current node.
 		currentNode = currentNodeWithDist.nextHop.Node.PubKeyBytes
+
+		if currentNode == target {
```

Comment:
comment from commit would be nice to have here

Reply:
Comment added. Not in this commit, but in the commit where it actually becomes important to check at the end.